src/main/java/org/repodriller/CommitVisitorPersistenceMechanisMap.java

```diff
@@ -0,0 +1,83 @@
+package org.repodriller;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.log4j.Logger;
+import org.repodriller.domain.Commit;
+import org.repodriller.persistence.PersistenceMechanism;
+import org.repodriller.scm.CommitVisitor;
+import org.repodriller.scm.SCMRepository;
+
+public class CommitVisitorPersistenceMechanisMap {

```

Comment:
Typo identified, thanks. The name is not intuitive, I was out of ideas. But  `CommitVisitor` is already the main Visitor interface name. What about `CommitVisitorIterator`, since methods initialize/processCommit/finalize iterate over its map?